src/views/MapView.vue

```diff
@@ -109,48 +109,36 @@ export default {
     },
     async reserveStreets() {
```

Comment:
These functions don't look like they have to be async. Might be a nice thing to remove to reduce the chance of unexpected behavior.